README.md

```diff
@@ -29,66 +16,73 @@ npm i git-guppy --save-dev
 
 ## Usage
 
-### Git integration
+### 1. Declaring hooks as gulp tasks
```

Comment:
**Motivation**: Initiating with the gulp task declaration - which is something already common for the user - makes it seam more easy to install.